### ISSUE TITLE
GRU for question 3

### ISSUE THREAD

**Issue by rawmarshmellows** [aunjgr/cs224n-2017#1]
Hi I've also been working on CS224n and I was wondering whether you were able to reduce the loss while running this command: python q3 gru.py predict -c gru while normalizing the state using the sigmoid function. I ask because I was unable to do this the loss just stays around 13 but if I don't normalize the state I am able to get the loss to converge to 0.

Oh and also I think you may have a typo in your code in your `q3_gru_cell.py`:
`o_t = tf.nn.tanh(tf.matmul(inputs, U_o) + r_t * tf.matmul(state, W_r) + tf.expand_dims(b_o, 0), name="o_t")`

The `W_r` should probably be `W_o`
